packages/web-components/tests/integration/ui/jest.config.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+process.env.JEST_PLAYWRIGHT_CONFIG = `${__dirname}/jest-playwright.config.js`;
+
+module.exports = {
+  globalSetup: '<rootDir>/tests/integration/ui/setup',
+  globalTeardown: '<rootDir>/tests/integration/ui/teardown',
+  setupFilesAfterEnv: ['expect-playwright', '<rootDir>/tests/integration/ui/setup-env'],
+  rootDir: '../../../',
+  roots: ['<rootDir>/src/components'],
+  testEnvironment: 'jest-playwright-preset',
+  testRunner: 'jest-circus/runner',
+  testRegex: '.*_steps\\.js$',
```

Comment:
Looks great Akira! One question, for the suffix `_steps.js` would these technically be considered e2e, or maybe `integration`? Just not sure if `steps` is as clear. We are following `ComponentName.e2e.js` as the filename structure for React, perhaps we can follow something similar?

Reply:
Thank you @jeffchew for your feedback. While UI integration tests are for partial integration rather than full integration as in E2E, you are right that there are some similarities.

`_steps` wording is from Cucumber, so I can see it sounds odd. If we use `.e2e.js` Jest finds WebDriverIO tests that would fail the test suite. That said, any other naming ideas...?

Reply:
Got it, maybe `.steps.js` instead of `_steps.js`? At least we are following convention with `*.test.js` and `*.e2e.js`?

Reply:
I see, makes sense - Thanks @jeffchew for the idea!

Reply:
Updated: https://github.com/carbon-design-system/ibm-dotcom-library/pull/3204/commits/996b5e7732d55f0403939c84fb8722a24ad94c5c